python/lsst/afw/geom/affineTransform.cc

```diff
@@ -81,8 +82,8 @@ PYBIND11_PLUGIN(_affineTransform) {
     clsAffineTransform.def("setParameterVector", &AffineTransform::setParameterVector);
     clsAffineTransform.def_static("makeScaling", (AffineTransform (*)(double)) &AffineTransform::makeScaling);
     clsAffineTransform.def_static("makeScaling", (AffineTransform (*)(double, double)) &AffineTransform::makeScaling);
-    clsAffineTransform.def_static("makeRotation", (AffineTransform (*)(Angle)) &AffineTransform::makeRotation);
-    clsAffineTransform.def_static("makeRotation", (AffineTransform (*)(Extent2D)) &AffineTransform::makeRotation);
+    clsAffineTransform.def_static("makeRotation", &AffineTransform::makeRotation, "angle"_a);
+    clsAffineTransform.def_static("makeTranslation", &AffineTransform::makeTranslation, "translation"_a);
```

Comment:
Hardly. The expression `(AffineTransform (*)(Extent2D)) &AffineTransform::makeRotation` is not evaluated by Pybind11, but by the compiler. My guess is that the rules for function pointer casts in C/C++ are quite loose.